core/src/main/java/com/graphhopper/routing/weighting/custom/EdgeToValueEntry.java

```diff
@@ -0,0 +1,32 @@
+/*
+ *  Licensed to GraphHopper GmbH under one or more contributor
+ *  license agreements. See the NOTICE file distributed with this work for
+ *  additional information regarding copyright ownership.
+ *
+ *  GraphHopper GmbH licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except in
+ *  compliance with the License. You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package com.graphhopper.routing.weighting.custom;
+
+import com.graphhopper.util.EdgeIteratorState;
+
+/**
+ * This internal interface calculates a double value from the specified EdgeIteratorState. Used for CustomWeighting.
+ *
+ * @see {{@link GeoToValueEntry}} or {@link EnumToValueEntry}
```

Comment:
Minor: there is a double `{{}}` here and I would either list all implementing subclasses or none of them? Probably none of them because this can change and its easy to find the subclasses.

Reply:
Ok